### DISCUSSION TITLE
Default oauth2-redirect.html content is invalid and out of date

### DISCUSSION THREAD

**Discussion by julienlavergne** [fastapi/fastapi#8873] [Questions]
### Description

The html content for oauth2-redirect.html is invalid and out of date. The lastest openapi version is correct.
As reported in this issue on swaggerui https://github.com/swagger-api/swagger-ui/issues/5111, the script tag must be moved inside the html tag.

Current content is here https://github.com/tiangolo/fastapi/blob/master/fastapi/openapi/docs.py#L107
New content is here https://github.com/swagger-api/swagger-ui/blob/master/dist/oauth2-redirect.html

I think it would be good to dynamically reference the latest version of oauth2-redirect.html included in swaggerui bundle instead of copying the content in this repo.

**Comment by mdgilene** [comment]
As a side note, because of this issue, you cannot authorize using the OpenIdConnect authorization_code flow. Since the redirect page doesn't function properly no auth code can be passed to the token endpoint. Probably affects other flows but haven't tested.

**Comment by YuriiMotov** [comment]
Seems that everything is Ok in current version
